### PR TITLE
EL-2663: Schema for format dropcaps

### PR DESCRIPTION
**Pull request by michaelguild13**

### CODE REVIEW

**Comment by michaelguild13** [comment]
Yes, it's unrelated to my changes. It has to do with the custom_embed

File: src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json",
+  "title": "Block properties",
+  "description": "Block properties for style formatting content elements",
+  "type": "object",
+  "properties": {
+    "dropcap": {
+      "title": "Dropcap",
+      "description": "Style the first letter or first word with a dropcap",
+      "type": "string",
+      "enum": ["letter", "word"]
```
Comment (yury-sannikov) [inline src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json]: I think per @awong7 we're not going to support `word` type

**Comment by yury-sannikov** [comment]
overall, this PR has not enough tests. It should have both positive and negative tests for each use case

**Comment by michaelguild13** [comment]
> overall, this PR has not enough tests. It should have both positive and negative tests for each use case

Sure thing. Wasn't fully aware of what tests were needed for this. I'll update accordingly.

File: src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json (hunk 1 above)
Comment (michaelguild13) [reply src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json]: @yury-sannikov in that case, should dropcap be a bool?

**Comment by michaelguild13** [comment]
@yury-sannikov I've updated the test to include bad cases as well

**Pull request merged**